CONTRIBUTING

```diff
@@ -0,0 +1,89 @@
+# Contributing to Dipy
```

Comment:
Fixed that other things as well. Thanks for taking a look!

On Mon, Jan 18, 2016 at 11:31 AM, Eleftherios Garyfallidis <
notifications@github.com> wrote:

> In CONTRIBUTING
> <https://github.com/nipy/dipy/pull/838#discussion_r50035912>:
>
> > @@ -0,0 +1,89 @@
> > +# Contributing to Dipy
>
> Thanx for starting to write such a document. Shouldn't this be an rst file
> in the dipy/doc folder?
> Here you wrote a markdown file but we don't have any markdown documents in
> dipy. I have a feeling this is in the wrong format and possibly folder.
> This information should be rendered by sphinx or something in the docs. You
> can also have a copy in the setup.py folder but that is secondary.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/838/files#r50035912>.
>